peps/pep-0813.rst

```diff
@@ -0,0 +1,196 @@
+PEP: 813
+Title: The Pretty Print Protocol
+Author: Barry Warsaw <barry@python.org>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 07-Nov-2025
+Python-Version: 3.15
+Post-History: Pending
+
+
+Abstract
+========
+
+This PEP describes the "pretty print protocol", a collection of changes proposed to make pretty printing more
+customizable and convenient.
+
+
+Motivation
+==========
+
+"Pretty printing" is a feature which provides a capability to format object representations for better
+readability.  The core functionality is implemented by the standard library :mod:`pprint`.  ``pprint``
+includes a class and APIs which users can invoke to format and print more readable representations of objects.
+Important use cases include pretty printing large dictionaries and other complicated objects.
```

Comment:
I'd say a key use case is interactive debugging, pretty-printing by default is a big reason I use IPython.